.changeset/blue-bobcats-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/preact": minor
+---
+
+Allows rendering lazy components
```

Comment:
Could you add a code snippet showcasing how to use lazy components? Apart from that, LGTM!

Reply:
There was another example in `examples/framework-preact`, but @matthewp asked for only one example

Reply:
There was another example in `examples/framework-preact`, but @matthewp  asked for only one example. Or do you want to write it at this file?

Reply:
I added snippet here and wrote, that fallback will be not displayed in SSR, may be do we need to add this information to docs?